File: LavaLipe/LavaLipe/Models/ClienteModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LavaLipe.Models
{
    public class ClienteModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string CPFCNPJ { get; set; }
        public string Phone { get; set; }
        public int Excluido { get; set; }

    }
}

File: FrontLavaLipe/src/Models/clienteModel.ts
export class clienteModel{
    id:number
    name:string;
    cpfcnpj: string; 
    phone: string;
    excluido:number;

    constructor() {
        this.id = 0;
        this.name ='';
        this.cpfcnpj ='';
        this.phone ='';
        this.excluido = 0;

      }
}